src/v/cluster/topics_frontend.cc

```diff
@@ -130,6 +113,53 @@ cluster::errc map_errc(std::error_code ec) {
     return errc::replication_error;
 }
 
+ss::future<std::vector<topic_result>> topics_frontend::update_topic_properties(
+  std::vector<topic_properties_update> updates,
+  model::timeout_clock::time_point timeout) {
+    auto leader = _leaders.local().get_leader(model::controller_ntp);
+
+    // no leader available
+    if (!leader) {
+        co_return create_topic_results(updates, errc::no_leader_controller);
+    }
+
+    // current node is a leader, just replicate
+    if (leader == _self) {
+        auto result = co_await _stm.invoke_on(
```

Comment:
Leave a comment that we need an empty batch to ensure that our log is up to date.